modules/Emails/include/ListView/ListViewDataEmailsSearchOnIMap.php

```diff
@@ -0,0 +1,316 @@
+<?php
```

Comment:
(class ListViewDataEmailsSearchOnIMap and class ListViewDataEmailsSearchOnCrm not completely refactored yet)